profiles/targets/desktop/package.use

```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Andreas Sturmlechner <asturm@gentoo.org> (2024-03-07)
+# Common flags needed by Qt6 revdeps
+dev-qt/qttools designer qdbus
```

Comment:
Why designer? Sound like a odd thing to need by default (see https://wiki.gentoo.org/wiki/Project:Qt/Qt6_migration_notes#From_qttools)

Note that many packages were depending on dev-qt/designer:5 because they needed Qt5UiTools (not designer itself), qttools:6 gives Qt6Uitools with USE=widgets regardless of designer. May be good to verify that things really need it before flipping the default.

Reply:
thanks, one more important thing to check.